Formula/lastpass-cli.rb

```diff
@@ -18,13 +18,27 @@ class LastpassCli < Formula
   depends_on "cmake" => :build
   depends_on "docbook-xsl" => :build
   depends_on "pkg-config" => :build
+  depends_on "curl" if MacOS.version >= :mojave
   depends_on "openssl"
   depends_on "pinentry" => :optional
 
   def install
     ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
-    system "make", "PREFIX=#{prefix}", "install"
-    system "make", "MANDIR=#{man}", "install-doc"
+
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_MANDIR:PATH=#{man}
+    ]
+
+    if MacOS.version >= :mojave
+      curl_prefix = Formula["curl"].opt_prefix
+      args << "-DCMAKE_PREFIX_PATH=#{curl_prefix}"
+    end
```

Comment:
Let's just delete the whole if block, since keg only deps' `opt_dir` gets put on the `CMAKE_PREFIX_PATH` through environment variables in the super env.

Reply:
Yep. Sounds like the right answer to me.